mm2src/coins/solana/solana_tests.rs

```diff
@@ -141,9 +220,7 @@ mod tests {
             .compat()
             .await
             .unwrap();
-        assert_eq!(valid_tx_details.total_amount, BigDecimal::from(0.0001));
-        assert_eq!(valid_tx_details.coin, "SOL".to_string());
-        assert_eq!(valid_tx_details.received_by_me, BigDecimal::from(0.0001));
```

Comment:
Why have you removed these lines?

Reply:
I've moved the same assertion in `solana_transaction_simulations`